modules/tensor_mechanics/tests/cp_slip_rate_integ/crysp_sinteg.i

```diff
@@ -0,0 +1,218 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 2
+  ny = 2
+  nz = 2
+  elem_type = HEX8
+  displacements = 'ux uy uz'
+[]
+
+[Variables]
+  [./ux]
+    block = 0
+  [../]
+  [./uy]
+    block = 0
+  [../]
+  [./uz]
+    block = 0
+  [../]
+[]
+
+[AuxVariables]
+  [./stress_zz]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+  [./fp_zz]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+  [./e_zz]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+  [./gss1]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+[]
+
+[Functions]
+  [./tdisp]
+    type = ParsedFunction
+    value = 0.01*t
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+    disp_z = uz
```

Comment:
Remove all disp_* parameters please, they are deprecated and not needed (set displacements in GlobalParams instead!)